### PR TITLE
disable processing of users moderation list when rendering a forum

### PR DESCRIPTION
**Pull request by mgreau**

### CODE REVIEW

File: forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl [hunk 1]
```diff
@@ -16,7 +16,26 @@
     for(moderator in moderators) {
       String userView = uicomponent.getScreenName(moderator);
       ++i;
+      int index = userView.lastIndexOf("/");
+      if (index != -1) {
+        String groupName = userView.substring(index+1);
+        groupName = groupName.substring(0, 1).toUpperCase() + groupName.substring(1);
+        if (!userView.contains(":")) {
+          userView = "Any in " + groupName;
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl]: Label hardcoded in english

File: forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl [hunk 2]
```diff
@@ -16,7 +16,26 @@
...
+      int index = userView.lastIndexOf("/");
+      if (index != -1) {
+        String groupName = userView.substring(index+1);
+        groupName = groupName.substring(0, 1).toUpperCase() + groupName.substring(1);
+        if (!userView.contains(":")) {
+          userView = "Any in " + groupName;
+        } else {
+          String membershipName = userView.substring(0, userView.indexOf(":"));
+          if (membershipName.equals("*")) membershipName = "Any";
+          userView = membershipName + " in " + groupName;
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl]: Label hardcoded in english

File: forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl [hunk 3]
```diff
@@ -16,7 +16,26 @@
...
+        }
   %>
+        <span>${userView}</span>
+    <%
+        if(total != i) {
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl]: This block is duplicated in both cases. It should be mutualized.

File: forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl (hunk 2 above)
Comment (wkhessairi) [inline forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl]: OK this should be updated

File: forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl (hunk 3 above)
Comment (wkhessairi) [inline forum/webapp/src/main/webapp/templates/forum/webui/UIForumModerator.gtmpl]: OK

File: forum/webapp/src/main/resources/locale/portlet/forum/ForumPortlet_en.properties [hunk 4]
```diff
@@ -322,13 +324,13 @@ UIViewTopic.action.DeleteTopic=Delete This Topic
 #										UIPostForm																						#
 #############################################################################
 UIPopupWindow.title.UIViewPost=View Post
-UIForumPopupWindow.title.ViewPost=#{UIPopupWindow.title.UIViewPost}
-UIForumChildPopupWindow.title.ViewPost=#{UIPopupWindow.title.UIViewPost}
-UIForumChildPopupWindow.title.UIViewPost=#{UIPopupWindow.title.UIViewPost}
+UIForumPopupWindow.title.ViewPost=#{UIPopupWindow.title.UIViewPost}
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/resources/locale/portlet/forum/ForumPortlet_en.properties]: There are a lot of unchanged lines in the PR. Problem of EOL characters ?

**Pull request closed without merging**